code.user.js

```diff
@@ -91,6 +92,23 @@
             this.inventoryUrlBase += '/';
     }
 
+    //#region language
+    var language = g_strLanguage;
```

Comment:
Can you check if it actually exists (such as typeof g_strLanguage !== 'undefined' && g_strLanguage != null ? g_strLanguage : 'en'). 
This is because all too frequently Steam is causing issues with undeclared variables.